java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -669,34 +641,37 @@ public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFilesForSp
 
         @Test
         public void atomicallyUpdateStatusToReadyForGCAndCreateNewActiveFileShouldFailIfFilesNotActive() throws StateStoreException {
```

Comment:
This test is broken. It looks like it's been made to assert the wrong behaviour, because the conditional expression has been removed from the update. We just need a different conditional expression.

Reply:
Fixed, thanks